### ISSUE TITLE
group_concat function doesn't work in lower case

### ISSUE THREAD

**Issue by kdabir** [agershun/alasql#1898]
when used as `GROUP_CONCAT` in query it works but when used `group_concat` it throws the following:

```
TypeError: alasql.fn["group_concat"] is not a function. (In 'alasql.fn["group_concat"](g['fieldName'])', 'alasql.fn["group_concat"]' is undefined)
```

this is not the case with other aggregation functions like `count` for example.

**Comment by vishal6557** [comment]
Can you assign to me, @mathiasrw  Thanks!

**Comment by Dedsec0098** [comment]
Hi I already have experience in fixing these issues. Can you please assign it to me?

**Comment by mathiasrw** [comment]
Fix released as part of v4.3.1
